lib/mayaUsdAPI/render.h

```diff
@@ -18,13 +18,16 @@
 
 #include <mayaUsdAPI/api.h>
 
+#include <string>
+
+#ifdef WANT_MATERIALX_BUILD
```

Comment:
Because this variable is used in an API header I needed to add it to the mayaUsdAPI.h file, so clients like LookdevX will get the correct build (since they don't set this var in their cmake files).